video/mp_image.c

```diff
@@ -549,6 +549,15 @@ void mp_image_copy_attributes(struct mp_image *dst, struct mp_image *src)
     assign_bufref(&dst->dovi_buf, src->dovi_buf);
     assign_bufref(&dst->film_grain, src->film_grain);
     assign_bufref(&dst->a53_cc, src->a53_cc);
+
+    for (int n = 0; n < dst->num_ff_side_data; n++)
+        av_buffer_unref(&dst->ff_side_data[n].buf);
+    MP_RESIZE_ARRAY(NULL, dst->ff_side_data, src->num_ff_side_data);
```

Comment:
I would add an empty line between the clearing and resize, to properly visually split these actions. Having such visual split might make sense for the resize and then finally copying the side data as well.

Reply:
Sure, added.

Reply:
Thanks. could you also put the sync of the `dst->num_ff_side_data = src->num_ff_side_data;` together with the MP_RESIZE_ARRAY? as those for me logically are the resize.

Otherwise 👍